Core/src/main/java/com/plotsquared/core/generator/HybridPlotWorld.java

```diff
@@ -204,7 +204,7 @@ public void loadConfiguration(ConfigurationSection config) {
                 }
                 Object value;
                 try {
-                    final boolean accessible = field.isAccessible();
+                    final boolean accessible = field.canAccess(field);
```

Comment:
This will throw an `IllegalArgumentException`. However, we do not need the accessible stuff at all: `getField` only returns public fields and those fields all belong to the current class or superclasses. `setAccessible` would only be needed for `private` fields from superclasses, but as we do not access them, this code isn't needed.

(Also, we wouldn't need to "reset" the accessible state of the field, as it is bound to the `Field` object rather than the underlying field, and the `Field` object isn't referenced outside of this method - it will be just garbage collected)

Reply:
See that's why I thought this whole thing looked funny hah. Basically this whole thing can be cleaned up and removed if I'm understanding correctly?

Reply:
Yes, only the `field.get(this)` call is needed.

Reply:
I'll get a new PR up for that :D